projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/b2b-saved-cart.ts

```diff
@@ -301,6 +301,87 @@ export function verifyCartDetails(cart: any) {
     });
 }
 
+export function getCartItem(name: string) {
+  return cy.get('cx-cart-item-list').contains('.cx-item-list-row', name);
+}
+
+export function restoreSavedCart(cart: any) {
+  const restoreSavedCartAlias = interceptRestoreSavedCartEndpoint(cart.code);
+  const getAllSavedCartAlias = interceptGetAllSavedCartEndpoint();
+
+  cy.get('cx-saved-cart-details-action .btn-primary').click();
+
+  cy.get('cx-saved-cart-form-dialog').within(() => {
+    cy.get('button[aria-label="Restore"]').click();
+  });
+
+  cy.wait(`@${restoreSavedCartAlias}`)
+    .its('response.statusCode')
+    .should('eq', 200);
+
+  alerts
+    .getSuccessAlert()
+    .should(
+      'contain',
+      `Existing cart is activated by ${cart.code} successfully`
+    );
+
+  cy.wait(`@${getAllSavedCartAlias}`)
+    .its('response.statusCode')
+    .should('eq', 200);
+
+  cy.get('cx-saved-cart-list .cx-saved-cart-list-no-saved-carts').should(
+    'contain',
+    'No Saved Carts Found'
+  );
+
+  // assert that restored cart became active cart
+  verifyMiniCartQuantity(1);
+
+  // assert that it is now in the cart page
+  visitCartPage();
+
+  cy.get('.cart-details-wrapper .cx-total').should(
+    'contain',
+    `Cart #${cart.code}`
+  );
+
+  verifyCartDetails(cart);
+}
+
+export function AddToActiveCart(cart: any, product: SampleProduct) {
+  cy.intercept(
+    'POST',
+    `${Cypress.env('OCC_PREFIX')}/${Cypress.env('BASE_SITE')}/${Cypress.env(
+      'OCC_PREFIX_USER_ENDPOINT'
+    )}/*/carts/*/entries?*lang=en&curr=USD`
+  ).as('add_to_cart');
+
+  cy.intercept(
+    'GET',
+    `${Cypress.env('OCC_PREFIX')}/${Cypress.env(
+      'BASE_SITE'
+    )}/cms/pages?pageType=ContentPage&pageLabelOrId=%2Fcart&lang=en&curr=USD`
+  ).as('cart_page');
```

Comment:
```suggestion
 function interceptCartPageEndpoint() {
      cy.intercept(
        'GET',
        `${Cypress.env('OCC_PREFIX')}/${Cypress.env(
          'BASE_SITE'
        )}/cms/pages?pageType=ContentPage&pageLabelOrId=%2Fcart&lang=en&curr=USD`
      ).as(CART_PAGE_ALIAS);
      
      return `@${CART_PAGE_ALIAS}`
  }
```

Is this function existing already in:  projects/storefrontapp-e2e-cypress/cypress/helpers/order-history.ts

Reply:
Yes, but I am not sure if I should import a function from order history to saved cart, if there's a place where we can add  such shared functions.. we can reuse the function